Answers.md

```diff
@@ -0,0 +1,9 @@
+If I were to describe semantic HTML, I would say that semantic HTML is a way of creating div's that have meaning. For example, a <p> is a stands for paragraph. It also allows code to be more readable by users and allows for better SEO.
+
+The differences between display:block and inline is that display block takes up one whole line and inline block only encases the content thats inside it.
+
+The for parts of the box model are. Content, Padding, Border, and Margin;
+
+When you use align-items in flexbox you are using the vertical axis.
+
```

Comment:
Yup initially. Now if you did something like `flex-direction: column` then you are using the horizontal axis. The main thing is you are using the cross-axis